### ISSUE TITLE
Engine: Jenkins Workflows

### ISSUE THREAD

**Issue by hoegertn** [taimos/projen-pipelines#11]

**Comment by thejames97** [comment]
I stumbled across your repos while looking for CDK CICD ideas for Jenkins.  I'm guessing this hasn't been implemented since I see no signs of it in the code.  I'd be happy to contribute it if I'm able to get it to work, but I would need some idea of what you had in mind for getting this work with Jenkins.

**Comment by hoegertn** [comment]
Hi, yes Jenkins has not been implemented yet. You can have a look at the GitHub and GitLab implementations. The goal would be to have a `Jenkinsfile` be generated that does the same.

**Comment by automartin5000** [comment]
I'd be an immediate user here. Not sure if I can contribute, but happy to help test

**Comment by automartin5000** [comment]
Actually, wouldn't the first step here be to create a Jenkins component for Projen just like GitHub and GitLab? And then apply the patterns in this repo?

**Comment by Lock128** [comment]
Good point, @automartin5000  - but we'd initially do this like we do it for CodeCatalyst and integrate the workflow generation directly in this project. We can later discuss about contributing this back to projen!
